srcs/python/kungfu/policy/policy.py

```diff
@@ -0,0 +1,19 @@
+class Policy():
```

Comment:
we could just put this in `srcs/python/kungfu/policy/__init__.py` and remove `srcs/python/kungfu/policy/policy.py`